addons/crm/models/crm_lead.py

```diff
@@ -875,6 +875,9 @@ def redirect_lead_opportunity_view(self):
 
     @api.model
     def get_empty_list_help(self, help):
+        if help:
+            return help
```

Comment:
Could be great to add a docstring to this get_empty_list_help to explain its new behavior